samples/AnomalyDetector/tests/Generated/Tests/AnomalyDetectorClientTests.cs

```diff
@@ -0,0 +1,740 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using AnomalyDetector;
+using AnomalyDetector.Models;
+using Azure;
+using Azure.Core;
+using Azure.Identity;
+using NUnit.Framework;
+
+namespace AnomalyDetector.Tests
+{
+    public partial class AnomalyDetectorClientTests : AnomalyDetectorTestBase
+    {
+        public AnomalyDetectorClientTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [Test]
+        [Ignore("Only validating compilation of test scaffoldings")]
+        public async Task DetectUnivariateEntireSeries_ShortVersion()
```

Comment:
What is the difference between these tests and ones in samples?

Reply:
Same but without response handling

Reply:
Why there is no response handling? Shouldn't there are some `Assert`s?

Reply:
We do not have the "expected values" to assert for the response values.
I could consider to add some assertions on the status code.